lib/dnsimple/default.rb

```diff
@@ -13,10 +13,10 @@ module Default
 
     class << self
 
-      # List of configurable keys for {Client}
-      # @return [Array] of option keys
-      def keys
```

Comment:
This is a breaking change. It seems a bit too much to include in a quite innocent Rubocop update. Can I ask you the reason of this change?

Reply:
Sure thing. New `Style/HashEachMethods` cop introduced was warning about having `.keys.each` to `.each_keys`. As `keys`is not an Enumerable (seems that Rubocop doesn't check that) we can't do that. So I took the path of actually renaming it instead of silencing the cop. Makes sense?

https://github.com/dnsimple/dnsimple-ruby/pull/174/files#diff-5798b0ae3c49cfe70adabe901e3ee44bR66

Reply:
As this is just Rubocop being dumb, can we silence it? I'm not sure I want to go through the hell of a deprecation because Rubocop can't figure out this is not a Hash.

Reply:
Silenced it 👍